db/Models/app.models.js

```diff
@@ -60,4 +60,49 @@ function fetchComments(review_id) {
   });
 }
 
-module.exports = { fetchCategories, fetchReviews, fetchReview, fetchComments };
+function addUser(username) {
+  return db
+    .query(
+      `
+    INSERT INTO users
+    (username, name, avatar_url)
+    VALUES
+    ($1, $2, $3)
+    RETURNING *
+`,
+      [
+        username,
+        "jason",
+        "https://avatars.githubusercontent.com/u/9919?s=460&v=4",
+      ]
+    )
+    .then(({ rows }) => {
+      return rows[0];
+    });
+}
+
+function addingComment(review_id, username, body) {
```

Comment:
This looks ok and seems like it works however it is extra to what the ticket asks for. addUser is not needed, and you should test the addingComment model with an existing user. Good code though